### PR TITLE
Add ability to set directory to store sessions

### PR DESCRIPTION
**Pull request by MojtabaTajik**
Now user can set directory to store sessions in there and not inside program's executable

### CODE REVIEW

File: TLSharp.Core/Session.cs [hunk 1]
```diff
@@ -14,9 +14,18 @@ public interface ISessionStore
 
     public class FileSessionStore : ISessionStore
     {
+        private readonly string _storePath;
+
+        public FileSessionStore(string storePath = "Sessions")
```
Comment (knocte) [inline TLSharp.Core/Session.cs]: you're providing a default value here which is different than what we had before

Comment (MojtabaTajik) [reply TLSharp.Core/Session.cs]: The default value added to backward compatibility to previous versions.

Comment (knocte) [reply TLSharp.Core/Session.cs]: I'm saying that this is not backwards compatible, the "Sessions" string was not there before.

Comment (MojtabaTajik) [reply TLSharp.Core/Session.cs]: @knocte Now it's compatible.

File: TLSharp.Core/Session.cs [hunk 2]
```diff
@@ -14,9 +14,18 @@ public interface ISessionStore
 
     public class FileSessionStore : ISessionStore
     {
+        private readonly string _storePath;
+
+        public FileSessionStore(string storePath = "")
```
Comment (knocte) [inline TLSharp.Core/Session.cs]: ok, now I prefer this to have a better type, it should be `DirectoryInfo`, not string. (And it should default to null.)

**Comment by knocte** [comment]
@MojtabaTajik I've finished refining your contribution and pushed the enhancement to master: https://github.com/sochix/TLSharp/commit/e4d0e97571a724c22aa8d519f4e66a1638aaa10e . Thanks!

**Pull request closed without merging**